packages/google_sign_in/google_sign_in_ios/darwin/Tests/GoogleSignInTests.m

```diff
@@ -567,6 +712,29 @@ - (void)testRefreshTokensUnknownError {
 
 #pragma mark - addScopes
 
+- (void)testRequestScopesPassesScopes {
```

Comment:
Mostly I didn't do test backfill in this PR, but I noticed we had a lot of tests of error conditions for `addScopes:...` but nothing that tested that we actually requested the right scopes, so added that quickly while I was here.